packages/react/etc/react.api.md

```diff
@@ -9379,21 +9379,26 @@ export interface IThemeSlotRule {
 // @public (undocumented)
 export interface ITimePickerProps extends Omit<IComboBoxProps, 'options' | 'selectedKey' | 'defaultSelectedKey' | 'multiSelect' | 'text' | 'defaultValue' | 'onChange'> {
     allowFreeform?: boolean;
+    dateAnchor?: Date;
     defaultValue?: Date;
     increments?: number;
     label?: string;
-    onChange?: (event: React_2.FormEvent<IComboBox>, time: Date) => void;
+    onChange?: (event?: React_2.FormEvent<IComboBox>, time?: Date) => void;
```

Comment:
This is an API breaking change: existing onChange listeners for TimePicker won't expect that `event` and `time` might be undefined, and will be broken by this change.

Can this API change be reverted?

Reply:
No problem at all, let me revert the onChange prop right now. Thank you and @sopranopillow for catching this.

Reply:
Reverted the `onChange` parameters back to their original typings.